bigquery/tests/unit/test_client.py

```diff
@@ -2876,6 +2876,49 @@ def test_insert_rows_json(self):
             path='/%s' % PATH,
             data=SENT)
 
+    def test_list_partitions(self):
+        from google.cloud.bigquery.table import Table
+
+        ROWS = 3
```

Comment:
These should not be constants (make lowercase). Many of the other tests do this, but it's something we're slowly fixing throughout the tests.